src/assets/ElmAsset.js

```diff
@@ -0,0 +1,43 @@
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+const process = require('process');
+
+// Currently no pretty way to install Elm tools on behalf of the user :/
+async function compile(name, options) {
+  try {
+    const elmCompiler = await localRequire('node-elm-compiler', name);
+    // compileToStringSync does not throw an Error, even when there is one.
+    // Other methods will throw an error, but do not bubble up to Parcel
+    const data = elmCompiler.compileToStringSync(name, options);
+    if (!data) {
+      throw 'You either have an error in your code, or are missing the Elm tools here https://guide.elm-lang.org/install.html.';
+    }
+    return data;
+  } catch (error) {
+    throw error;
```

Comment:
Why cat the error at all if you are just going to throw it again?

Reply:
Sorry, almost all of these are updated in a local branch with your concerns. I was hopeful that the WIP label made it obvious that this code is trash atm.